src/main/cms/cms_menu_imu.c

```diff
@@ -326,7 +326,9 @@ static const OSD_Entry cmsx_menuSimplifiedTuningEntries[] =
     { "I GAIN",         OME_FLOAT, NULL, &(OSD_FLOAT_t) { &cmsx_simplified_i_gain,            SIMPLIFIED_TUNING_MIN, SIMPLIFIED_TUNING_MAX, 1, 10 }, 0 },
     { "PD RATIO",       OME_FLOAT, NULL, &(OSD_FLOAT_t) { &cmsx_simplified_pd_ratio,          SIMPLIFIED_TUNING_MIN, SIMPLIFIED_TUNING_MAX, 1, 10 }, 0 },
     { "PD GAIN",        OME_FLOAT, NULL, &(OSD_FLOAT_t) { &cmsx_simplified_pd_gain,           SIMPLIFIED_TUNING_MIN, SIMPLIFIED_TUNING_MAX, 1, 10 }, 0 },
+#ifdef USE_D_MIN
```

Comment:
While this is correct, this is only a little part of the fix - if D_MIN is not used, then all the manipulation of and access to the `simplified_dmin_ratio` should be removed.

Reply:
yeah, thats only cosmetic, will remove the logic too. ill send an update later